src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -437,6 +491,35 @@ private BuildData MergeBuildManifests(List<BuildData> buildsMetadata)
             return mergedBuild;
         }
 
+        private SigningInformation MergeSigningInfo(List<SigningInformation> signignInformation)
+        {
+            SigningInformation mergedInfo = signignInformation[0];
```

Comment:
Can this list ever be empty so we need to guard before this access?

Reply:
Yeah, the check is been done in line 96 `if (signingInformation.Any())`